templates/default/tests/markdownFeatures.spec.js

```diff
@@ -0,0 +1,136 @@
+const { test, expect } = require('@playwright/test');
+const { MarkdownPage } = require('./markdown-page');
+
+test.describe.parallel('wiki links', () => {
+  test.beforeEach(async ({ page }) => {
+    const Page = new MarkdownPage(page);
+    await Page.goto('/test/fixtures/markdownFeatures');
+    await Page.getData();
+  });
+
+  test('parses a wiki link', async ({ page }) => {
+    const link = page.locator('#wikiLink > p > a');
+    await expect(link).toHaveClass('internal');
+    await link.click();
+    await expect(page).toHaveURL('/test/fixtures/fixturepage');
+  });
+
+  test('parses a wiki link with custom divider', async ({ page }) => {
+    const link = page.locator('#wikiLink-custom > p > a');
+    await expect(link).toHaveClass('internal');
+    await link.click();
+    await expect(page).toHaveURL('/test/fixtures/fixturepage');
+  });
```

Comment:
These two tests do exactly the same thing. The second one should also check if the text inside of the link is the custom one (written after `|`).